src/pages/marketplace/details-market.vue

```diff
@@ -149,7 +149,8 @@ export default {
       try {
         this.showLoading()
         const market = await this.$store.$marketplaceApi.getMarketplaceById({ marketId: this.marketId })
-        const authorities = await this.$store.$marketplaceApi.getAuthoritiesByMarketplace({ marketId: this.marketId })
+        const palletId = this.$store.$marketplaceApi.palletId
+        const authorities = await this.$store.$marketplaceApi.getAuthoritiesByMarketplace({ palletId, marketId: this.marketId })
```

Comment:
$marketPlaceApi should handler palletId automatically

Reply:
Got it, I'm going to chance this. 👍🏼